globalvars.py

```diff
@@ -129,9 +129,72 @@ class GlobalVars:
 
     api_request_lock = threading.Lock()
 
-    num_posts_scanned = 0
-    post_scan_time = 0
-    posts_scan_stats_lock = threading.Lock()
+    class post_scan_stat:  # noqa: N801
```

Comment:
Python class names are `CamelCase`, `snake_case` is for variable names

Reply:
Let's see if flake8 is happy now :-P

Reply:
Err... I did not notice the "View changes" button and conducted a forced push after renaming the class. Looks like your commits disappeared. Sorry. (@ArtOfCode- )

Reply:
I never had any commits here...

Reply:
![image](https://user-images.githubusercontent.com/65436504/82367207-70f4bc80-99e1-11ea-8568-f74488a8adde.png)
I suppose that means you have made some changes to those source code? (Anyways now flake8 is happy, and I am waiting for additional reviews and then move on to commit code sitting on my computer for next inner class.)